<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1249</width>
    <height>593</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <italic>true</italic>
    <stylestrategy>PreferAntialias</stylestrategy>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QTabWidget" name="Tab">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="tabLogs">
           <attribute name="title">
            <string>Логи</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <widget class="QLabel" name="label">
                <property name="font">
                 <font>
                  <family>Segoe Print</family>
                  <pointsize>14</pointsize>
                  <italic>true</italic>
                  <bold>false</bold>
                  <stylestrategy>PreferAntialias</stylestrategy>
                 </font>
                </property>
                <property name="text">
                 <string>Логи</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QScrollArea" name="scrollArea">
                <property name="widgetResizable">
                 <bool>true</bool>
                </property>
                <widget class="QWidget" name="scrollAreaWidgetContents">
                 <property name="geometry">
                  <rect>
                   <x>0</x>
                   <y>0</y>
                   <width>969</width>
                   <height>437</height>
                  </rect>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                   <widget class="QListView" name="object_listView"/>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tabStateAndSetting">
           <attribute name="title">
            <string>Состояние и настройки</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QTabWidget" name="tabWidget">
                <property name="currentIndex">
                 <number>0</number>
                </property>
                <widget class="QWidget" name="Controller">
                 <attribute name="icon">
                  <iconset theme="preferences-desktop">
                   <normaloff>.</normaloff>.</iconset>
                 </attribute>
                 <attribute name="title">
                  <string>Информация о контролерах</string>
                 </attribute>
                 <layout class="QHBoxLayout" name="horizontalLayout_3">
                  <item>
                   <widget class="QTableView" name="tableViewController"/>
                  </item>
                 </layout>
                </widget>
                <widget class="QWidget" name="Systeam">
                 <attribute name="title">
                  <string>Система</string>
                 </attribute>
                 <layout class="QHBoxLayout" name="horizontalLayout_4">
                  <item>
                   <layout class="QVBoxLayout" name="verticalLayout_6">
                    <item>
                     <widget class="QTableView" name="tableCredentialServerData"/>
                    </item>
                    <item>
                     <widget class="QTableView" name="tableViewAutoStart"/>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_2">
                    <item>
                     <widget class="QTableView" name="tableComPort"/>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Элементы управления</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QPushButton" name="runServerButton">
             <property name="text">
              <string>Запустить сервер</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="endEmulatorButton">
             <property name="text">
              <string>Завершить опрос контролеров</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnWSServerStart">
             <property name="text">
              <string>Websoket Server</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>Отправить клиентам сообщение</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="openDoorButton">
             <property name="text">
              <string>Открыть двери</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1249</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
